samples/batch/batch-ingestion-client/PowerBI/guide.md

```diff
@@ -1,13 +1,13 @@
 
-# Visualizing Outputs of the Accelerator in Power BI
+# Visualizing the transcription output in Power BI
 
 ## Setup Guide
 
-The following guide will help you create Power BI reports from templates that will help to realize value from the speech transcription accelerator you just deployed.
+The following guide will help you create Power BI reports from templates that will help to realize value from the speech transcription out.
```

Comment:
2 "Will help" in the same sentences make it difficult to read. :-)

Reply:
good point. address